File: All-Services/ClientApp/src/Model/Admin/Equipment.ts
import { EquipmentType } from "./EquipmentType";
import { Manufacturer } from "./Manufacturer";

export class Equipment {
  constructor(
    public id?: number,
    public model?: string,
    public equipmentTypeId?: number,
    public manufacturerId?: number,
    public equipmentType?: EquipmentType,
    public manufacturer?: Manufacturer
  ) { }
}

File: AllServicesData/Model/Equipment.cs
namespace AllServicesData.Model
{
    public class Equipment : EntityBase
    {
        public string Model { get; set; }
        public int EquipmentTypeId { get; set; }
        public int ManufacturerId { get; set; }
        public virtual EquipmentType EquipmentType { get; set; }
        public virtual Manufacturer Manufacturer { get; set; }
    }
}
